packages/shared/src/toDisplayString.ts

```diff
@@ -27,7 +27,7 @@ export const toDisplayString = (val: unknown): string => {
 
 const replacer = (_key: string, val: any): any => {
   // can't use isRef here since @vue/shared has no deps
-  if (val && val.__v_isRef) {
+  if (val?.__v_isRef) {
```

Comment:
The compilation result using the `optional chain operator (?.)` is very different from the compilation result using the `short-circuit operator (&&)` directly, and I'm not sure if there is any impact on the size of the framework if it has been used a lot? Of course it's actually not that pessimistic hahaha.

---

[Babel Playground](https://babeljs.io/repl#?browsers=defaults%2C%20not%20ie%2011%2C%20not%20ie_mob%2011&build=&builtIns=false&corejs=3.21&spec=false&loose=false&code_lz=MYewdgzgLgBCDWMC8MDeBfAUKSsQAcoBLcAQwBsAFAJwOTngH4A6YbcaOQksCgYQrl6CFqQAUASna4YEABYhqUPkWrAArkWWDh8ZqRgAyQw32TMQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3%2Ctypescript&prettier=false&targets=&version=7.19.0&externalPlugins=&assumptions=%7B%7D)

![image](https://user-images.githubusercontent.com/51586637/188561725-8fe6e4bd-bd83-4a2b-995f-cb0958c4cdf4.png)

Reply:
You can open the `FileSize` comparison on the left to see.

`(&&) ---> 70 - 53 = 17 bytes`

![image](https://user-images.githubusercontent.com/51586637/188566724-ff75955b-7ba6-4312-926c-6f36254d2918.png)

---

`(?.) ---> 98 - 43 = 55 bytes `

![image](https://user-images.githubusercontent.com/51586637/188566786-c2b8b956-f78c-47cc-ad03-968156b1a187.png)